packages/bot/src/components/saveload-modal.jsx

```diff
@@ -0,0 +1,197 @@
+import                       '../assets/sass/google-drive.scss';
+
+const initial_option = { is_local: true, save_as_collection: true };
+
+const SaveLoadModal = ({
+    button_status,
+    handleFileChange,
+    is_authorised,
+    is_save_modal,
+    is_saveload_modal_open,
+    onConfirmSave,
+    onDriveConnect,
+    onLoadClick,
+    toggleSaveLoadModal,
+}) => {
+    const title = translate(is_save_modal ? 'Save Bot' : 'Load Bot');
+
+    return (
+        <Modal
+            title={title}
+            className='modal--saveload'
+            is_open={is_saveload_modal_open}
+            toggleModal={toggleSaveLoadModal}
+        >
+            <Formik
+                initialValues={initial_option}
+                onSubmit={
+                    is_save_modal ?
+                        (values => onConfirmSave(values)) :
+                        (values => onLoadClick(values))
+                }
+            >
+                {
+                    ({ values: { is_local, save_as_collection }, setFieldValue }) => (
+                        <Form>
+                            <div className='modal__content'>
+                                <div className='modal__content-row'>
+                                    <RadioGroup
+                                        className='radio-group__saveload-type'
+                                        name='is_local'
+                                        items={[
+                                            {
+                                                id   : 'local',
+                                                label: <IconRadio
+                                                    text={translate('Local')}
+                                                    icon={<ToolbarLocalIcon />}
+                                                />,
+                                                value: true,
+                                            },
+                                            {
+                                                id   : 'Drive',
+                                                label: <IconRadio
+                                                    text={translate('Google Drive')}
+                                                    icon={<ToolbarDriveIcon />}
+                                                    google_drive_connected={is_authorised}
+                                                    onDriveConnect={onDriveConnect}
+                                                />,
+                                                value    : false,
+                                                disabled : !is_authorised,
+                                                className: is_authorised ? '' : 'dc-radio-group__item-disabled',
+                                            },
+                                        ]}
+                                        selected={is_authorised ? is_local : true}
```

Comment:
```suggestion
                                        selected={is_authorised && !is_local}
```

Reply:
The logic is not correct. If `is_authorised`, i wan to return `is_local` else always return `true`. In your suggestion, if `is_authorised` is `true`, it will return revery of `is_local`

Reply:
i'll change to `is_authorised || is_local`. Thanks